src/utils/FourWay.js

```diff
@@ -368,13 +368,14 @@ class FourWay {
         info.layoutSize = source.getLayoutSize();
         info.settingsArray = (await this.read(eepromOffset, info.layoutSize)).params;
         info.settings = Convert.arrayToSettingsObject(info.settingsArray, info.layout);
-
+        
         /**
          * If not AM32, then very likely BLHeli_32, even if not - we can't
          * handle it.
          */
         const validNames = source.getValidNames();
-        if(!validNames.includes(info.settings.NAME)) {
+
+        if(info.settings.TYPE !== 'AM32' && !validNames.includes(info.settings.NAME)) {
```

Comment:
This will fail on each AM32 ESC which is not flashed with exactly this version, we are thus locking out everyone that does not have this version running yet.

Added on top that the `validName` check is currently not properly working on multiple devices and needs to be reverted or improves to check for the bootloader byte instead (as suggested by @AlkaMotors ).

When adding EEPROM values it is important to also extend the layout and bump the layout revision accordingly otherwise we don't know how to interpret the values exactly and end up in a state where older versions don't have this expected field, we are still trying to interpret the data somehow.